mixins/brand.js

```diff
@@ -85,7 +85,7 @@ export default {
 
     return {
       bodyAttrs: { class: cssClass },
-      title:     this.$store.getters['i18n/t']('nav.title'),
+      title:     getVendor(),
```

Comment:
Could we make this:

`title: brandMeta?.vendor || getVendor()`

and then add a vendor of 'SUSE Rancher' to the metadata.json for the suse theme in assets/brand ?

Reply:
That would be a quick way of getting 'SUSE Rancher' into the title, but all the other places `getVendor` is used would still show plain 'Rancher'
I was thinking of using `setVendor` and putting the check for `metadata.vendor` [here](https://github.com/rancher/dashboard/blob/master/store/index.js#L499) where the ui-pl setting is found and applied, like: 

```
    const pl = res.settings?.find(x => x.name === 'ui-pl')?.value;
    const brand = res.settings?.find(x => x.name === SETTING.BRAND)?.value;

    if ( pl && pl !== SETTING.PL_RANCHER_VALUE ) {
      setVendor(pl);
    } else if (brand) {
      let brandMeta;

      try {
        brandMeta = require(`~/assets/brand/${ brand }/metadata.json`);
      } catch {
      }
      if (brandMeta.vendor) {
        setVendor(brandMeta.vendor);
      }
    }
```
...plus some setting/reverting on the support page with setVendor when a subscription is added or removed

Reply:
Okay - let's just get this in as is, and revisit for 2.6.X